### PR TITLE
Proofreading

### PR DESCRIPTION
**Pull request by agitter**
These are primarily minor edits noticed during proofreading.  The missing reference separator led to inconsistent reference numbering.  I'll open a Manubot issue to discuss that.  The updated citation tag now cites the research paper instead of the correction to that paper.

### CODE REVIEW

File: content/02.intro.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ Similarly, for continuous outcomes, linear regression can be seen as a single-la
...
 In a famous and early example, scientists from Google demonstrated that a neural network "discovered" that cats, faces, and pedestrians were important components of online videos [@url:http://research.google.com/archive/unsupervised_icml2012.html] without being told to look for them.
-What if, more generally, deep learning take advantage of the growth of data in biomedicine to tackle challenges in this field? Could these algorithms identify the "cats" hidden in our data---the patterns unknown to the researcher---and suggest ways to act on them? In this review, we examine deep learning's application to biomedical science and discuss the unique challenges that biomedical data pose for deep learning methods.
+What if, more generally, deep learning takes advantage of the growth of data in biomedicine to tackle challenges in this field? Could these algorithms identify the "cats" hidden in our data---the patterns unknown to the researcher---and suggest ways to act on them? In this review, we examine deep learning's application to biomedical science and discuss the unique challenges that biomedical data pose for deep learning methods.
```
Comment (cgreene) [inline content/02.intro.md]: 👍

File: content/02.intro.md [hunk 2]
```diff
@@ -112,7 +112,7 @@ In oncology, current "gold standard" approaches include histology, which require
 One example is the PAM50 approach to classifying breast cancer where the expression of 50 marker genes divides breast cancer patients into four subtypes.
 Substantial heterogeneity still remains within these four subtypes [@doi:10.1200/JCO.2008.18.1370; @doi:10.1158/1078-0432.CCR-13-0583].
 Given the increasing wealth of molecular data available, a more comprehensive subtyping seems possible.
-Several studies have used deep learning methods to better categorize breast cancer patients: For instance, denoising autoencoders, an unsupervised approach, can be used to cluster breast cancer patients [@doi:10.1142/9789814644730_0014], and CNN can help count mitotic divisions, a feature that is highly correlated with disease outcome in histological images [@doi:10.1007/978-3-642-40763-5_51].
+Several studies have used deep learning methods to better categorize breast cancer patients: For instance, denoising autoencoders, an unsupervised approach, can be used to cluster breast cancer patients [@doi:10.1142/9789814644730_0014], and CNNs can help count mitotic divisions, a feature that is highly correlated with disease outcome in histological images [@doi:10.1007/978-3-642-40763-5_51].
```
Comment (cgreene) [inline content/02.intro.md]: 👍

File: content/03.categorize.md [hunk 3]
```diff
@@ -12,7 +12,7 @@ In spite of such optimism, the ability of deep learning models to indiscriminate
 Imagine a deep neural network is provided with clinical test results gleaned from electronic health records.
 Because physicians may order certain tests based on their suspected diagnosis, a deep neural network may learn to "diagnose" patients simply based on the tests that are ordered.
 For some objective functions, such as predicting an International Classification of Diseases (ICD) code, this may offer good performance even though it does not provide insight into the underlying disease beyond physician activity.
-This challenge is not unique to deep learning approaches; however, it is important for practitioners to be aware of these challenges and the possibility in this domain of constructing highly predictive classifiers of questionable actual utility.
+This challenge is not unique to deep learning approaches; however, it is important for practitioners to be aware of these challenges and the possibility in this domain of constructing highly predictive classifiers of questionable utility.
```
Comment (cgreene) [inline content/03.categorize.md]: 👍

File: content/03.categorize.md [hunk 4]
```diff
@@ -48,30 +48,28 @@ The technique of reusing features from a different task falls into the broader a
 Though we've mentioned numerous successes for the transfer of natural image features to new tasks, we expect that a lower proportion of negative results have been published.
 The analysis of magnetic resonance images (MRIs) is also faced with the challenge of small training sets.
 In this domain, Amit et al. [@tag:Amit2017_breast_mri] investigated the tradeoff between pre-trained models from a different domain and a small CNN trained only with MRI images.
-In contrast with the other selected literature, they found a smaller network trained with data augmentation on few hundred images from a few dozen patients can outperform a pre-trained out-of-domain classifier.
+In contrast with the other selected literature, they found a smaller network trained with data augmentation on a few hundred images from a few dozen patients can outperform a pre-trained out-of-domain classifier.
```
Comment (cgreene) [inline content/03.categorize.md]: 👍

File: content/03.categorize.md [hunk 5]
```diff
@@ -48,30 +48,28 @@ The technique of reusing features from a different task falls into the broader a
...
 Shin et al. [@tag:Shin2016_cad_tl] compared various deep network architectures, dataset characteristics, and training procedures for computer tomography-based (CT) abnormality detection.
 They concluded that networks as deep as 22 layers could be useful for 3D data, despite the limited size of training datasets.
 However, they noted that choice of architecture, parameter setting, and model fine-tuning needed is very problem- and dataset-specific.
 Moreover, this type of task often depends on both lesion localization and appearance, which poses challenges for CNN-based approaches.
 Straightforward attempts to capture useful information from full-size images in all three dimensions simultaneously via standard neural network architectures were computationally unfeasible.
-Instead, two-dimensional models were used to either process image slices individually
-(2D), or aggregate information from a number of 2D projections in the native space (2.5D).
+Instead, two-dimensional models were used to either process image slices individually (2D) or aggregate information from a number of 2D projections in the native space (2.5D).
```
Comment (cgreene) [inline content/03.categorize.md]: 👍

File: content/03.categorize.md [hunk 6]
```diff
@@ -48,30 +48,28 @@ The technique of reusing features from a different task falls into the broader a
...
 
 Roth et al. compared 2D, 2.5D, and 3D CNNs on a number of tasks for computer-aided detection from CT scans and showed that 2.5D CNNs performed comparably well to 3D analogs, while requiring much less training time, especially on augmented training sets [@tag:Roth2015_view_agg_cad].
 Another advantage of 2D and 2.5D networks is the wider availability of pre-trained models.
-But reducing the dimensionality is not always helpful.
-Nie et al.
-[@tag:Nie2016_3d_survival] showed that multimodal, multi-channel 3D deep architecture was successful at learning high-level brain tumor appearance features jointly from MRI, functional MRI, and diffusion MRI images, outperforming single-modality or 2D models.
+However, reducing the dimensionality is not always helpful.
```
Comment (cgreene) [inline content/03.categorize.md]: 👍

File: content/03.categorize.md [hunk 7]
```diff
@@ -48,30 +48,28 @@ The technique of reusing features from a different task falls into the broader a
...
 Overall, the variety of modalities, properties and sizes of training sets, the dimensionality of input, and the importance of end goals in medical image analysis are provoking a development of specialized deep neural network architectures, training and validation protocols, and input representations that are not characteristic of widely-studied natural images.
 
 Predictions from deep neural networks can be evaluated for use in workflows that also incorporate human experts.
-In a large dataset of mammography images, Kooi et al. [@tag:Kooi2016_mamm_lesions] demonstrated that deep neural networks outperform the traditional computer-aided diagnosis system at low sensitivity and perform comparably at high sensitivity.
+In a large dataset of mammography images, Kooi et al. [@tag:Kooi2016_mamm_lesions] demonstrated that deep neural networks outperform a traditional computer-aided diagnosis system at low sensitivity and perform comparably at high sensitivity.
```
Comment (cgreene) [inline content/03.categorize.md]: 👍

File: content/03.categorize.md [hunk 8]
```diff
@@ -48,30 +48,28 @@ The technique of reusing features from a different task falls into the broader a
...
 They also compared network performance to certified screening radiologists on a patch level and found no significant difference between the network and the readers.
 However, using deep methods for clinical practice is challenged by the difficulty of assigning a level of confidence to each prediction.
 Leibig et al. [@tag:Leibig2016_dr] estimated the uncertainty of deep networks for diabetic retinopathy diagnosis by linking dropout networks with approximate Bayesian inference.
-Techniques that assign confidences to each prediction should aid pathologist-computer interactions and improve uptake by physicians.
+Techniques that assign confidences to each prediction should aid physician-computer interactions and improve uptake by physicians.
```
Comment (cgreene) [inline content/03.categorize.md]: 👍

File: content/03.categorize.md [hunk 9]
```diff
@@ -102,7 +100,7 @@ In this case, they applied a series of natural language processing (NLP) techniq
 They first extracted all diseases mentioned in the reports using a state-of-the-art NLP tool, then applied a new method, NegBio [@arxiv:1712.05898], to filter negative and equivocal findings in the reports.
 Evaluation on four independent datasets demonstrated that NegBio is highly accurate for detecting negative and equivocal findings (~90% in F₁ score, which balances precision and recall [@doi:10.1038/nmeth.3945]).
 The resulting dataset [@url:https://nihcc.app.box.com/v/ChestXray-NIHCC] consisted of 112,120 frontal-view chest X-ray images from 30,805 patients, and each image was associated with one or more *text-mined* (weakly-labeled) pathology categories (e.g. pneumonia and cardiomegaly) or "no finding" otherwise.
-Further, Wang et al. [@arxiv:1705.02315] used this dataset with a unified weakly-supervised multi-label image classification framework, to detect common thoracic diseases.
+Further, Wang et al. [@arxiv:1705.02315] used this dataset with a unified weakly-supervised multi-label image classification framework to detect common thoracic diseases.
```
Comment (cgreene) [inline content/03.categorize.md]: 👍

File: content/03.categorize.md [hunk 10]
```diff
@@ -143,32 +141,32 @@ Some studies focused on jointly extracting biomedical entities and relations sim
...
 Jiang et al. proposed a biomedical domain-specific word embedding model to reduce the manual labor of designing semantic representation for the same task [@doi:10.1504/IJDMB.2016.074878].
 Gu et al. employed a maximum entropy model and a CNN model for chemical-induced disease relation extraction at the inter- and intra-sentence level, respectively [@doi:10.1093/database/bax024].
-For drug-drug interaction, Zhao et al. used a CNN that employs word embeddings with the syntactic information of a sentence as well as features of part-of-speech tags and dependency trees [@doi:10.1093/bioinformatics/btw486].
-Asada et al. experimented with an attention CNN [@doi:10.18653/v1/w17-2302], and Yi et al. a RNN model with multiple attention layers [@arxiv:1705.03261].
+For drug-drug interactions, Zhao et al. used a CNN that employs word embeddings with the syntactic information of a sentence as well as features of part-of-speech tags and dependency trees [@doi:10.1093/bioinformatics/btw486].
```
Comment (cgreene) [inline content/03.categorize.md]: 👍

File: content/03.categorize.md [hunk 11]
```diff
@@ -143,32 +141,32 @@ Some studies focused on jointly extracting biomedical entities and relations sim
...
+Asada et al. experimented with an attention CNN [@doi:10.18653/v1/w17-2302], and Yi et al. proposed an RNN model with multiple attention layers [@arxiv:1705.03261].
 In both cases, it is a single model with attention mechanism, which allows the decoder to focus on different parts of the source sentence.
 As a result, it does not require dependency parsing or training multiple models.
 Both attention CNN and RNN have comparable results, but the CNN model has an advantage in that it can be easily computed in parallel, hence making it faster with recent graphics processing units (GPUs).
 
-For biotopes event extraction, Li et al. employed CNN and distributed representation [@doi:10.18653/v1/w16-3012] while Mehryary et al. used long short-term memory (LSTM) networks to extract complicated relations [@doi:10.18653/v1/w16-3009].
+For biotopes event extraction, Li et al. employed CNNs and distributed representation [@doi:10.18653/v1/w16-3012] while Mehryary et al. used long short-term memory (LSTM) networks to extract complicated relations [@doi:10.18653/v1/w16-3009].
```
Comment (cgreene) [inline content/03.categorize.md]: 👍

File: content/03.categorize.md [hunk 12]
```diff
@@ -189,25 +187,23 @@ It is difficult for us to provide a strong statement on the broad utility of the
...
-Jagannatha and Yu [@pmcid:PMC5119627] further employed a bidirectional LSTM structure to extract adverse drug events from electronic health records, and Lin et al. [@doi:10.18653/v1/w17-2341] investigated using CNN to extract temporal relations.
-While promising, a lack of rigorous evaluations of the real-world utility of these kinds of features makes current contributions in this area difficult to evaluate.
+Jagannatha and Yu [@pmcid:PMC5119627] further employed a bidirectional LSTM structure to extract adverse drug events from electronic health records, and Lin et al. [@doi:10.18653/v1/w17-2341] investigated using CNNs to extract temporal relations.
+While promising, a lack of rigorous evaluation of the real-world utility of these kinds of features makes current contributions in this area difficult to evaluate.
```
Comment (cgreene) [inline content/03.categorize.md]: 👍 above here (stopped doing each individual change)

File: content/03.categorize.md [hunk 13]
```diff
@@ -330,11 +326,10 @@ There are approaches to protect models, but they pose their own challenges.
...
-Both Beaulieu-Jones et al. and Esteban et al. train models on synthetic data generated under differentially private and observe performance from a transfer learning evaluation that is only slightly below models trained on the original, real data.
+Both Beaulieu-Jones et al. and Esteban et al. train models on synthetic data generated under differential privacy and observe performance from a transfer learning evaluation that is only slightly below models trained on the original, real data.
 Taken together, these results suggest that differentially private GANs may be an attractive way to generate sharable datasets for downstream reanalysis.
 
-Federated learning [@url:http://proceedings.mlr.press/v54/mcmahan17a.html] and secure aggregations [@url:https://eprint.iacr.org/2017/281.pdf
-@tag:Papernot2017_pate] are complementary approaches that reinforce differential privacy.
+Federated learning [@url:http://proceedings.mlr.press/v54/mcmahan17a.html] and secure aggregations [@url:https://eprint.iacr.org/2017/281.pdf; @tag:Papernot2017_pate] are complementary approaches that reinforce differential privacy.
```
Comment (cgreene) [inline content/03.categorize.md]: 👍 the above changes in this file

File: content/04.study.md [hunk 14]
```diff
@@ -387,7 +387,7 @@ They achieved impressive performance, even for cell types where the subset perce
 However, they did not benchmark against random forests, which tend to work better for imbalanced data, and their data was relatively low dimensional.
 
 Neural networks can also learn low-dimensional representations of single-cell gene expression data for visualization, clustering, and other tasks.
-Both scvis [@doi:10.1101/178624] and scVI [@arxiv:1709.02082] are unsupervised approaches based on VAEs.
+Both scvis [@doi:10.1101/178624] and scVI [@arxiv:1709.02082] are unsupervised approaches based on variational autoencoders  (VAEs).
```
Comment (cgreene) [inline content/04.study.md]: 👍 for the changes on this file

File: content/05.treat.md [hunk 15]
```diff
@@ -1,6 +1,6 @@
 ## The impact of deep learning in treating disease and developing new treatments
 
-Given the need to make better, faster interventions at the point of care---incorporating the complex calculus of a patients symptoms, diagnostics, and life history---there have been many attempts to apply deep learning to patient treatment.
+Given the need to make better, faster interventions at the point of care---incorporating the complex calculus of a patient's symptoms, diagnostics, and life history---there have been many attempts to apply deep learning to patient treatment.
```
Comment (cgreene) [inline content/05.treat.md]: 👍

File: content/06.discussion.md [hunk 16]
```diff
@@ -198,7 +198,7 @@ The contribution scores were then used to identify key phrases from a model trai
 Interpretation of embedded or latent space features learned through generative unsupervised models can reveal underlying patterns otherwise masked in the original input.
 Embedded feature interpretation has been emphasized mostly in image and text based applications [@tag:Radford_dcgan; @tag:Word2Vec], but applications to genomic and biomedical domains are increasing.
 
-For example, Way and Greene trained a variational autoencoder (VAE) on gene expression from The Cancer Genome Atlas (TCGA) [@doi:10.1038/ng.2764] and use latent space arithmetic to rapidly isolate and interpret gene expression features descriptive of high grade serous ovarian cancer subtypes [@tag:WayGreene2017_tybalt].
+For example, Way and Greene trained a variational autoencoder on gene expression from The Cancer Genome Atlas (TCGA) [@doi:10.1038/ng.2764] and use latent space arithmetic to rapidly isolate and interpret gene expression features descriptive of high grade serous ovarian cancer subtypes [@tag:WayGreene2017_tybalt].
```
Comment (cgreene) [inline content/06.discussion.md]: VAE was defined above - should we just use VAE here?

**Comment by cgreene** [comment]
By the way, I'm not sure if this counts for our iteration of the deep-review or if we should rely on @evancofer and @stephenra to review & merge. What do you think?

File: content/06.discussion.md (hunk 16 above)
Comment (agitter) [reply content/06.discussion.md]: Yes, I'll change to VAE.  I primarily wanted to move the acronym definition to the first use.

**Comment by agitter** [comment]
Let's let the new maintainers finish the review and merge when they approve.

**Comment by cgreene** [comment]
Ok - sounds good! Probably important to note that this one should probably be a regular merge instead of a squash merge due to the multi-contributor commits 😁

**Comment by agitter** [comment]
I wonder whether the merge commit in a squash merge would retain the co-authoring?  The [GitHub guide](https://help.github.com/articles/creating-a-commit-with-multiple-authors/) isn't clear.  The `Co-authored-by:` keyword is retained in the commit message of the merge commit, but I don't know whether it needs to be at the beginning or end of the commit message.

**Comment by evancofer** [comment]
LGTM

**Comment by stephenra** [comment]
LGMT, as well.

**Pull request merged**